src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,356 @@
+  --pf-c-data-toolbar__content--md--PaddingLeft: var(--pf-global--spacer--md);
+
+  @media screen and (max-width: $pf-global--breakpoint--md) {
+    --pf-c-data-toolbar__content--PaddingRight: var(--pf-c-data-toolbar__content--md--PaddingRight);
+    --pf-c-data-toolbar__content--PaddingLeft: var(--pf-c-data-toolbar__content--md--PaddingLeft);
+  }
+
+  // Expandable content
+  --pf-c-data-toolbar__expandable-content--PaddingTop: var(--pf-c-data-toolbar__content--PaddingTop);
+  --pf-c-data-toolbar__expandable-content--PaddingRight: var(--pf-c-data-toolbar__content--PaddingRight);
+  --pf-c-data-toolbar__expandable-content--PaddingBottom: var(--pf-c-data-toolbar__content--PaddingBottom);
+  --pf-c-data-toolbar__expandable-content--PaddingLeft: var(--pf-c-data-toolbar__content--PaddingLeft);
+  --pf-c-data-toolbar__expandable-content--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-data-toolbar__expandable-content--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+
+  // Expanded
+  --pf-c-data-toolbar__expandable-content--m-expanded--GridGap: var(--pf-global--gutter);
+
+  // Base spacer - shared value
+  --pf-c-data-toolbar--spacer--base: var(--pf-global--spacer--md);
+
+  // Spacer values
+  --pf-c-data-toolbar__item--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--spacer: var(--pf-c-data-toolbar--spacer--base);
+
+  // Item
+  --pf-c-data-toolbar--item--Display: flex;
+  --pf-c-data-toolbar--item--Visibility: visible;
+
+  // Toggle group
+  --pf-c-data-toolbar__group--m-toggle-group--spacer: var(--pf-global--spacer--sm);
+  --pf-c-data-toolbar__group--m-toggle-group--m-reveal--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__toggle--c-badge: var(--pf-global--spacer--sm);
+
+  // Button group
+  --pf-c-data-toolbar__group--m-button-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-button-group--space-items: var(--pf-global--spacer--sm);
+
+  // Icon button group spacer
+  --pf-c-data-toolbar__group--m-icon-button-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-icon-button-group--space-items: 0;
+
+  // Icon group spacer
+  --pf-c-data-toolbar__group--m-filter-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-filter-group--space-items: 0;
+
+  // Condensed spacer
+  --pf-c-data-toolbar__group--m-condensed--space-items: 0;
+
+  // Bulk select
+  --pf-c-data-toolbar--m-bulk-select--spacer: var(--pf-global--spacer--lg);
+
+  // Overflow menu item
+  --pf-c-data-toolbar--m-overflow-menu--spacer: var(--pf-c-data-toolbar--spacer--base);
+
+  // Label
+  --pf-c-data-toolbar--m-label--spacer: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar--m-label--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Search filter
+  --pf-c-data-toolbar--m-search-filter--spacer: var(--pf-global--spacer--sm);
+  --pf-c-data-toolbar--m-search-filter--space-items: 0;
+
+  // Separator
```

Comment:
Can we use the divider component here?

Reply:
I don't think so, we don't have a vertical divider option yet...

Reply:
Oh sorry, I meant adding that as an enhancement to that component and using it here. We didn't create the vertical divider because we didn't need that variation yet, but I think we do now. I'll create an issue for that and we can use this as it is and update it once we add the variation to the divider.